### ISSUE TITLE
Why are PODs not mentioned in the spec?

### ISSUE THREAD

**Issue by aviflax** [solid/solid-spec#118]
I’m totally new to Solid but it’s confusing to me that PODs seem totally crucial and foundational in some places (e.g. [How It Works](https://solid.inrupt.com/how-it-works)) but are entirely absent in other places — such as the spec. What’s the deal?

**Comment by RubenVerborgh** [comment]
Thanks, you're raising a very interesting point.

When we write "server" in the specification, this is to be understood as equivalent to "data pod". However, it is indeed confusing that the language is not aligned.

**Comment by RubenVerborgh** [comment]
> Isn't a server a collection of pods, one per account?

Right, good point. As usual, the word "server" has different levels of interpretation, ranging from the machine, to the HTTP interface, to individual software instances responding on different ports.

In the Solid specification, "server" seems to be, abstractly, the interface the client interacts with, which would be closer to the notion of (an interface to) a "pod". Whereas in the implementation of nodes-solid-server, the "server" is a collection of multiple "pods".

So probably the word "server" in the spec is not precise enough and might need to be "pod" or something else.

**Comment by melvincarvalho** [comment]
Tim suggested that a pod is a data folder with no parent.  Pod is one of those loose terms that is quite hard to define, but you know one when you see it.  The issue is that your data can live anywhere, on an origin, in a sub directory, on localhost, even on a usb stick.  

Do we actually benefit from defining it in the spec?  If not, we could put in some loose wording.

**Comment by RubenVerborgh** [comment]
The issue is more that the server talks about "server" in a confusing way. That definition of pod as a parentless folder wouldn't be helpful for the purposes of the spec, where it needs to be defined from an interface perspective.

**Comment by melvincarvalho** [comment]
It might be just me, but I've noticed increasingly a fascination with the term "POD".  Props to @sandhawke for coming up with that name.  Here's an example from the technical blog infoQ

> Solid starts with the concept of a POD (Personal Online Data stores). Within a Solid POD, users store information such as photos, comments, contacts, calendars, fitness, and health data. The POD may get located within a home, at work, or within a selected POD provider. It should be easy to move data at any time without interruption of services.

https://www.infoq.com/news/2018/10/solid-identity-platform

It's possibly tempting to just let everyone define it the way they want to, and they might anyway!
